ui/app/pages/swaps/select-quote-popover/select-quote-popover-constants.js

```diff
@@ -6,7 +6,7 @@ export const QUOTE_DATA_ROWS_PROPTYPES_SHAPE = PropTypes.shape({
   destinationTokenDecimals: PropTypes.number.isRequired,
   destinationTokenSymbol: PropTypes.string.isRequired,
   destinationTokenValue: PropTypes.string.isRequired,
-  isBestQuote: PropTypes.bool.isRequired,
+  isBestQuote: PropTypes.bool,
```

Comment:
I got complaints that `isBestQuote` was `undefined` in the console. For our usage, that'll practically always be equivalent to `false`.